webapp/src/main/webapp/iaf/gui/views/Webservices.html

```diff
@@ -39,7 +39,7 @@ <h4>Available ApiListeners</h4>
 									<td>{{::apiListener.method}}</td>
 									<td ng-if="::apiListener.error">{{::apiListener.error}}</td>
 									<td ng-if="::apiListener.schemaResource" class="hideBtn1170">
-										<a target="_blank" href="{{::rootURL}}api/{{::apiListener.uriPattern}}/{{::apiListener.schemaResource}}" class="btn btn-xs pull-right default" type="button"><i class="fa fa-arrow-circle-o-down aria-hidden="true"></i> <span>OpenAPI 3.0</span></a>
+										<a target="_blank" href="{{::rootURL}}api{{::apiListener.uriPattern}}/{{::apiListener.schemaResource}}" class="btn btn-xs pull-right default" type="button"><i class="fa fa-arrow-circle-o-down aria-hidden="true"></i> <span>OpenAPI 3.0</span></a>
```

Comment:
You can start the href with ``/`` instead of with ``{{::rootURL}}``

Reply:
rootURL contains the instance name for example /iaf-example/